Python/Video to PDF(CLI)/README.md

```diff
@@ -0,0 +1,36 @@
+# Video to PDF
+
+The aim of this script is to convert a given video into a pdf.
+This script:
+
+1. Takes a video as input.
+2. Extracts audio from it.
+3. Converts the audio into text.
+4. Writes this text into a <strong>PDF</strong> and saves it.
+
+## Setup Instructions
+
+You will need the following 3 modules to be installed:
+- Speech recognition
+- MoviePy
+- FPDF
+
```

Comment:
Add the command through which a user can install all the packages in a single go.